code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -48,6 +48,11 @@
 		overlay_x = 18, \
 		overlay_y = 12)
 
+/obj/item/gun/energy/pulse/carbine/lethal/Initialize(mapload)
+	. = ..()
+	select_fire()
+	select_fire()
```

Comment:
Why double?

Reply:
1 goes from pulse to stun, 2nd goes from stun to lethals

Reply:
At least comment what you're doing then